### PR TITLE
59 address performance issues

### PR DESCRIPTION
**Pull request by johnxussw**
Performance
- [x] Create a component | an emulated Zendesk button
    - [x] Click the real Zendesk button after the script loaded
- [x] Remove duplicated analytic script and npm packages

Accessibility
- [x] Change MegaMenu main menu tag from `<a>` to `<div>`
    - [x] Change MegaMenu source code
    - [x] Change the copy of the code in this solution
- [x] Heading tags should be used in descending order without skip
- [x] Change from `<html>` to `<html lang="en">`

### CODE REVIEW

**Comment by wicksipedia** [comment]
🐛 BUG - you have to click on the zendesk button twice to get the chat up

**Comment by wicksipedia** [comment]
changes to the mega menu should be done on that repo as well - we have a copy of the code as a workaround for getting it up and running

**Comment by tkapa** [comment]
Seems a shame that we aren't seeing much improvement on the Performance score :/

**Comment by tkapa** [comment]
🐛 Looks like we still have to click twice on the Zendesk chat to open it up (However I think that's because of the delayed load)

File: components/zendeskButton/zendeskButton.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        clearTimeout(clickEvent.current);
+        console.log("detect >>> ", zendeskNativeButton);
+        if (zendeskNativeButton) {
+          zendeskNativeButton.click();
+        }
+      }
+
+      const repeat = () => {
+        clickButton();
+        console.log("click", maxTrials.current);
```
Comment (tkapa) [inline components/zendeskButton/zendeskButton.tsx:41]: This should be removed when we're happy with the behaviour.
```suggestion
```

**Comment by wicksipedia** [comment]
🐛 layout issue

image

**Comment by johnxussw** [comment]
> Seems a shame that we aren't seeing much improvement on the Performance score :/

True. It only improves a little bit :(
The main bundle is extremely large.

**Comment by wicksipedia** [comment]
now the menu loads all the sections on page load
image

**Comment by johnxussw** [comment]
We used class "unstyled" a lot in MegaMenu, but we do not define it in MegaMenu source code.
It's confusing.

**Comment by johnxussw** [comment]
> now the menu loads all the sections on page load image

Done - fixed

**Review by bradystroud — APPROVED** [review]
lgtm ⚡️

**Pull request merged**